app/assets/stylesheets/application.css.scss

```diff
@@ -518,4 +518,19 @@ span.twitter-typeahead {
 
 .paper-preview-not-available {
   text-align: center;
+}
+
+.latest-run {
+  font-size: 0;
```

Comment:
Properties should be ordered border, border-radius, display, font-size, height, line-height, vertical-align, width